microsoft/testsuites/core/hv_module.py

```diff
@@ -118,6 +118,20 @@ def verify_initrd_modules(self, environment: Environment) -> None:
         ):
             missing_modules.remove("hid_hyperv")
 
+        # Fedora 41+ optimizes initrd by excluding non-essential modules like hv_netvsc
+        # Network drivers can be loaded later during network service initialization
+        node = environment.nodes[0]
+        if (
+            isinstance(node.os, Fedora)
+            and node.os.information.version.major >= 41
+            and "hv_netvsc" in missing_modules
+        ):
+            missing_modules.remove("hv_netvsc")
```

Comment:
this is the network module which should exist in the distro, either build-in or as a module

Reply:
Got it, Will revert back the changes for test case and update the PR